### DISCUSSION TITLE
How to change SFX?

### DISCUSSION THREAD

**Discussion by ItsJaxer** [Techokami/SonicWorldsNext#104] [Q&A]
This question probably makes it obvious that I'm a newcomer to Godot, but how am I supposed to replace sound effects with new ones?

**Comment by Techokami** [comment]
Well, the sound effects are stored in the Audio folder of the project root. Just go in there and replace the WAV files? I don't think you'll need to manually regenerate the corresponding .input file.

**Reply by ItsJaxer** [reply]
I tried that and it doesn't work

**Reply by ItsJaxer** [reply]
Correction, it actually works, just not if the .WAV extension is capitalized

**Reply by Techokami** [reply]
Yeah filenames are case-sensitive.

**Reply by ItsJaxer** [reply]
Nevermind, now it stopped working

**Reply by Renhoex** [reply]
There's a bunch of audio nodes inside the player nodes, have you been able to open the player scene and drag and drop your sounds into the sound stream as needed?
